plugins/item_tasks/web_client/stylesheets/selectTaskWidget.styl

```diff
@@ -0,0 +1,16 @@
+.g-task-custom-target-container
```

Comment:
This class name doesn't make much sense to me. I would expect it to start with `.g-select-task-` at least.